doc/manual/configuration.rst

```diff
@@ -774,6 +774,8 @@ packages that are built from a library: a ``-target`` packet that has the
 shared libraries needed during runtime and a ``-dev`` packet that has the
 header files and other needed files to link with this library.
 
+Bob does not support multiple or nested multiPackages in a single recipe, yet.
```

Comment:
"multiple" "multiPackage"-Entries are are not possible with yaml. That's the same as having multiple "packageScript"s. I think that can be left out. What is indeed not possible is a nested `multiPackage`:

```yaml
multiPackage:
    foo:
        multiPackage:
            bar:
                packageScript: ...
```

This is not implemented and will be rejected during schema validation. With some effort it could be implemented, though.